heartbeat/exportfs

```diff
@@ -417,6 +417,10 @@ testdir() {
 }
 exportfs_validate_all ()
 {
+	if ! echo "$OCF_RESKEY_fsid" | grep -q -E "^[A-Za-z0-9-]*$"; then
+		ocf_exit_reason "$OCF_RESKEY_fsid is not a valid fsid"
+		return $OCF_ERR_CONFIGURED
+	fi
```

Comment:
Missed that there's a check only for multiple directories. Anyway, exports(5) says here:
              Other filesystems can be identified with a small integer,  or  a
              UUID  which  should contain 32 hex digits and arbitrary punctua‐
              tion.
That doesn't match the regexp in grep above.